### ISSUE TITLE
"Select serial port" dropdown can be misleading when used with p5 web editor

### ISSUE THREAD

**Issue by kaganjd** [vanevery/p5.serialcontrol#14]
Using the serial control app alone, without the p5 web editor, you have to follow these steps to get data in the app's serial console:
1. select the serial port from the dropdown in the app that's connected to your Arduino, and
2. check the "Serial Console Enabled" box

If you're using the web editor, you can:
1. open the app
3. check the "Serial Console Enabled" box
2. connect to the app in your p5 sketch
4. "play" your sketch

You never have to select your serial port from the dropdown. Since you set the portName in the p5 sketch, you're getting data from the Arduino, but the serial port dropdown will show the first option in the dropdown, which may not be the port you're actually getting data from. 

<img src="https://user-images.githubusercontent.com/9204835/31139027-50efd42a-a83e-11e7-9760-d2e6431dc3ac.png" width="600px"></img>

One fix would be to figure out how to get the serial control app to display the port that's actually being read from, in cases where it's set from a p5 sketch instead of the dropdown menu.

**Comment by vanevery** [comment]
I agree, it would be nice for the serial control app to display the currently open port.

One of the issues is that the selected port in the drop down gets reset is that the sketch calls a "serial.list()" which repopulates the drop down.
